modules/base-realms/realm-google/idp-google.tf

```diff
@@ -0,0 +1,36 @@
+
+resource "keycloak_realm" "realm" {
+  realm   = "my-realm"
+  enabled = true
+}
+
+resource "keycloak_oidc_google_identity_provider" "google" {
+  realm       = module.realm.id
+  client_id     = var.client_id
+  client_secret = var.client_secret
+
+  sync_mode   = "FORCE"
+
+  # Defaults to empty, no OTP checking
+  # post_broker_login_flow_alias  = ""
+
+
+  # realm         = keycloak_realm.realm.id
+  # client_id     = var.google_identity_provider_client_id
+  # client_secret = var.google_identity_provider_client_secret
+  trust_email   = true
+  hosted_domain = "example.com"
+  
+  # These may need to be updated
+  # default_scopes = "openid profile email"
```

Comment:
I am playing around with the scopes a bit, still not 100% sure if we can remove profile from the user consent, seems to come with the openid scope in google